App/Sagas/DeepLinkSagas.ts

```diff
@@ -29,6 +30,10 @@ export function * inviteAfterOnboard () {
 }
 
 export function * routeThreadInvite(url: string, hash: string ) {
+  const reduxStarted: boolean = yield select(startupSelectors.started)
+  if (!reduxStarted) {
+    yield take(getType(StartupActions.startup))
+  }
```

Comment:
So basically, this function is currently being called before redux persist rehydration is complete. This logic blocks the saga until the startup/rehydration is complete.